test/test_mssql_server.py

```diff
@@ -27,38 +27,38 @@ def test_sql_server_cte():
     assert (
         Parser(
             """
-                    WITH x AS (
-                        SELECT * FROM n
-                    )
-                    SELECT
-                        *
-                    FROM x
-                    JOIN y ON x.a = y.a
-                        """
+                        WITH x AS (
+                            SELECT * FROM n
+                        )
+                        SELECT
+                            *
+                        FROM x
+                        JOIN y ON x.a = y.a
+                            """
         ).tables
         == ["n", "y"]
     )
 
     assert (
         Parser(
             """
-                    WITH foo AS (
-                        SELECT * FROM n
-                    )
-                    UPDATE z from foo set z.q = foo.y
-                        """
+                        WITH foo AS (
+                            SELECT * FROM n
+                        )
+                        UPDATE z from foo set z.q = foo.y
+                            """
         ).tables
         == ["n", "z"]
     )
 
     assert (
         Parser(
             """
-                    WITH foo AS (
-                         SELECT * FROM tab
-                    )
-                    DELETE FROM z JOIN foo ON z.a = foo.a
-                        """.strip()
+                        WITH foo AS (
+                             SELECT * FROM tab
+                        )
+                        DELETE FROM z JOIN foo ON z.a = foo.a
+                            """.strip()
```

Comment:
@collerek - I believe your IDE keeps adding these tabs here on each commit 😉

Reply:
Sneaky PyCharm! 🤣

Reply:
ok, I think black didn't like the `strip()` which was unnecessary anyway